packages/@uppy/google-drive/src/index.js

```diff
@@ -103,14 +116,49 @@ module.exports = class GoogleDrive extends Plugin {
   }
 
   getItemModifiedDate (item) {
-    return item.modifiedByMeDate
+    return item.modifiedTime
   }
 
   getItemThumbnailUrl (item) {
     return `${this.opts.serverUrl}/${this.GoogleDrive.id}/thumbnail/${this.getItemRequestPath(item)}`
   }
 
   render (state) {
+    // If the user has access to any Team Drives, handle them as needed.
+    if (state.plugins[this.id].hasTeamDrives) {
```

Comment:
You can do `this.getPluginState()` instead of accessing `state.plugins[this.id]`, is a bit clearer I think :)